src/lib.rs

```diff
@@ -196,15 +197,15 @@ impl ModDef {
     pub fn emit(&self) -> String {
         // self.validate();
         let mut emitted_module_names = IndexMap::new();
-        let mut mod_defs_as_strings = Vec::new();
-        self.emit_recursive(&mut emitted_module_names, &mut mod_defs_as_strings);
-        mod_defs_as_strings.join("\n")
+        let mut file = VastFile::new(VastFileType::Verilog);
```

Comment:
You wouldn't prefer SV syntax? Could set this to SV if you would prefer that.

Reply:
Nice, didn't realize that - updated the code.